Modules/Core/Common/include/itkVersor.hxx

```diff
@@ -140,14 +140,6 @@ Versor<T>::operator/(const Self & v) const
   return result;
 }
 
-/** Comparison operator */
```

Comment:
This is just removed, not replaced by the new macro. Is this intentional?

Reply:
Thanks for reviewing so carefully, but yes, it's intended 😸 The macro call defines `operator!=`as a simple inline one-liner, in the `h` file. So the `operator!=` implementation may just be removed from the corresponding `hxx` file.